docs/paper.md

```diff
@@ -125,7 +129,9 @@ Table: A **curated selection** of the metrics, tools and statistical tests curre
 
 ## Use in Academic Work
 
-In 2015, the Australian Bureau of Meteorology began developing a new verification system called Jive, which became operational in 2022. For a description of Jive see @loveday2024jive. The Jive verification metrics have been used to support several publications [@Griffiths:2017; @Foley:2020; @Taggart:2022d; @Taggart:2022b; @Taggart:2022c]. `scores` has arisen from the Jive verification system and was created to modularise the Jive verification functions and make them available as an open source package. `scores` also includes additional metrics that Jive does not contain.
+In 2015, the Australian Bureau of Meteorology began developing a new verification system called Jive, which became operational in 2022. For a description of Jive see @loveday2024jive. The Jive verification metrics have been used to support several publications [@Griffiths:2017; @Foley:2020; @Taggart:2022d; @Taggart:2022b; @Taggart:2022c]. 
+
+`scores` has arisen from the Jive verification system and provides Jive verification functions as a modular, open source package. `scores` also includes additional metrics that Jive does not contain.
```

Comment:
Note: the diff isn't showing it, but the sentence "`scores` has arisen from the Jive verification system and provides Jive verification functions as a modular, open source package." is modified from the original version.

Reply:
I'm happy with the new version.